build.gradle

```diff
@@ -927,7 +928,7 @@ project('standalone') {
                                  "-Dlog.dir=PRAVEGA_APP_HOME/logs",
                                  "-Dlog.name=pravega",
                                  "-Xdebug",
```

Comment:
According to: https://www.baeldung.com/java-application-remote-debugging  it is now prefered to set debug and debug options together like:
```
-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=127.0.0.1:8000
```

Reply:
Thanks for this reference, will prefer this debug option.